app/assets/javascripts/modules/moj.emailPreview.js

```diff
@@ -13,7 +13,20 @@
     },
 
     handleClick: function(event) {
-      event.target.search = $(this.el).serialize();
+      event.preventDefault();
+
+      var ajaxOptions = {
+        url:    event.target.href,
+        method: 'POST',
```

Comment:
Ah, the hidden method explains it, worth adding a comment? Otherwise is not clear how this actually works